website/docs/cloud-docs/projects/manage.mdx

```diff
@@ -0,0 +1,120 @@
+---
+page_title: Manage projects in HCP Terraform and Terraform Enterprise
+description: |-
+  Use projects to organize and group workspaces and create ownership boundaries
+  across your infrastructure.
+---
+
+# Manage projects
+
+This topic describes how to create and manage projects in HCP Terraform and Terraform Enterprise. A project is a folder containing one or more workspaces.
+
+## Requirements
+
+You must have the following permissions to manage projects:
+
+- You must be a member of a team with the **Manage all Projects**  permissions enabled to create a project. Refer to [Organization Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#organization-permissions) for additional information.
```

Comment:
When mentioning specific permissions I feel like we usually do quotes? I don't love it, but bold doesn't feel right _either_ because we aren't talking about something in UI element here.
```suggestion
- You must be a member of a team with the "Manage all Projects" permission to create a project. Refer to [Organization Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#organization-permissions) for additional information.
```

Reply:
This is referring to a UI element. There is a radio button in the interface with this label.